test/topologies.jl

```diff
@@ -492,14 +500,19 @@ end
   # more challenging case with inconsistent orientation
   e = connect.([(4, 1, 5), (2, 6, 4), (3, 5, 6), (4, 5, 6)])
   t = HalfEdgeTopology(e)
+  test_halfedge(e, t)
   n = collect(elements(t))
-  @test n == connect.([(5, 4, 1), (6, 2, 4), (6, 5, 3), (4, 5, 6)])
+  @test n == connect.([(4, 1, 5), (4, 6, 2), (6, 5, 3), (4, 5, 6)])
+
+  e = connect.([(1, 2, 3), (1, 3, 4), (2, 5, 3), (5, 4, 6), (3, 5, 4)])
+  t = HalfEdgeTopology(e)
+  test_halfedge(e, t)
 
   # indexable api
   g = GridTopology(10, 10)
   t = convert(HalfEdgeTopology, g)
-  @test t[begin] == connect((13, 12, 1, 2), Quadrangle)
-  @test t[end] == connect((110, 121, 120, 109), Quadrangle)
+  @test t[begin] == g[begin]
```

Comment:
```suggestion
  @test t[begin] == connect((1, 2, 13, 12), Quadrangle)
```

Reply:
My thought was that directly comparing to `g` would be more in line with testing that the edges indices in GridTopology match that of the converted HalfEdgeTopology. Currently the first element is the only one with matching edges/indices. The other two (10 and the last element) do not match the initial index of the corresponding elements in `g`. I don't have an opinion or preference, but you had mentioned that matching GridTopology was a goal.

Reply:
I noticed that even before the PR we didn't have guarantees to match the order with GridTopology. Since the resulting order is arbitrary, I think it makes more sense to hard-code it in the tests. That way we can quickly sketch the grid on paper and debug based on the test values.